src/libs/data-table-versions.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { parseGsUri } from 'src/components/data/data-utils'
+import { Ajax } from 'src/libs/ajax'
+import { getUser } from 'src/libs/auth'
+import { notify } from 'src/libs/notifications'
+import { useCancellation } from 'src/libs/react-utils'
+import * as Utils from 'src/libs/utils'
+
+
+export const dataTableVersionsRoot = '.data-table-versions'
```

Comment:
This is already a long name, so use your own judgment, but I might call this `dataTableVersionsPathRoot` so as not to confuse it with a css selector.

Reply:
Renamed in https://github.com/DataBiosphere/terra-ui/pull/3325/commits/b6c0e80be3f92a53372f3f4e776e4e965f95ce28.